src/main/iohook.js

```diff
@@ -254,6 +254,82 @@ const tilt = async () => {
 
 }
 
+
+
+const resizeOnADS = async () => {
+	
+	const ADSSize = Number.parseInt( preferences.value( 'actions.ADSSize' ) )
+	const resizeOnADSOptions = Number.parseInt( preferences.value( 'actions.resizeOnADS' ) )
+	const oldCrosshairSize = Number.parseInt( preferences.value( 'crosshair.size' ) )
+
+	log.info( 'Setting: Resize on ADS' )
+	await iohook.importIoHook()
+
+	if (resizeOnADSOptions === 1) {
+		iohook.hook.on( 'mousedown', event => {
+
+			if ( event.button === 2 ) {
```

Comment:
~Both of these functions are using `event.button === 2`, I think one or the other is incorrect~

I just realized what you're doing, disregard this